incubator/java-openliberty/image/project/pom.xml

```diff
@@ -24,7 +24,7 @@
         <!-- OpenLiberty runtime -->
         <liberty.groupId>io.openliberty</liberty.groupId>
         <liberty.artifactId>openliberty-runtime</liberty.artifactId>
-        <version.openliberty-runtime>19.0.0.12</version.openliberty-runtime>
+        <version.openliberty-runtime>{{.stack.liberty_version}}</version.openliberty-runtime>
         <version.liberty-maven-plugin>3.1</version.liberty-maven-plugin>
```

Comment:
That's good.. what about also:
1. The two usages in the 'enforcer' plugin:

>                         <regex>19.0.0.12</regex>
>                            <regexMessage>OpenLiberty runtime version must be 19.0.0.12</regexMessage>

 2.  A rephrasing of the README.md doc:  
`This stack is based on OpenJDK with container-optimizations in OpenJ9 and `Open Liberty v19.0.0.12`

Reply:
Missed those.  Will add.